examples/datasets/plot_shifted_dataset_regression.py

```diff
@@ -0,0 +1,89 @@
+"""
+Plot dataset source domain and shifted target domain
+====================================================
+
+This illustrates the :func:`~skada.datasets.make_shifted_dataset`
+dataset generator. Each method consists of generating source data
+and shifted target data. We illustrate here:
+covariate shift, target shift, concept drift, and sample bias.
+See detailed description of each shift in [1]_.
+
+.. [1] Moreno-Torres, J. G., Raeder, T., Alaiz-Rodriguez,
+       R., Chawla, N. V., and Herrera, F. (2012).
+       A unifying view on dataset shift in classification.
+       Pattern recognition, 45(1):521-530.
+"""
+# %% Imports
+
+import matplotlib.pyplot as plt
+
+from skada.datasets import make_shifted_datasets
+from skada import source_target_split
+
+
+# %% Helper function
+
+
+
+def plot_shifted_dataset(shift, random_state=42):
+    """Plot source and shifted target data for a given type of shift.
+
+    The possible shifts are 'covariate_shift', 'target_shift' or
+    'concept_drift'.
+
+    We use here the same random seed for multiple calls to
+    ensure same distributions.
+    """
+    X, y, sample_domain = make_shifted_datasets(
+        n_samples_source=20,
+        n_samples_target=20,
+        shift=shift,
+        noise=0.3,
+        label="regression",
+        random_state=random_state,
+    )
+    X_source, X_target, y_source, y_target = source_target_split(X, y, sample_domain=sample_domain)
+
+    print(y.shape, y_source.shape, y_target.shape, sample_domain.shape)
+    print()
+    fig, (ax1, ax2) = plt.subplots(1, 2, sharex="row", sharey="row", figsize=(8, 4))
+    fig.suptitle(shift.replace("_", " ").title(), fontsize=14)
+    plt.subplots_adjust(bottom=0.15)
+    s = ax1.scatter(
+        X_source[:, 0],
+        X_source[:, 1],
+        c=y_source*10,
+        vmax=1,
+        alpha=0.5,
+    )
+    cb=fig.colorbar(s)
```

Comment:
I think one colorbar should be enough per line

Reply:
it's done